alerts_consumer.go

```diff
@@ -141,13 +168,29 @@ func (c *AlertsConsumer) encodeMessage(fields map[string]interface{}) ([]byte, [
 			}
 			pt = sfxclient.Cumulative(route.Series, dims, counterVal)
 			pt.Timestamp = timestamp
+			if cwWhitelisted {
+				dat = &cloudwatch.MetricDatum{
+					MetricName: &route.Series,
+					Dimensions: cwDims,
+					Value:      aws.Float64(float64(counterVal)),
+					Timestamp:  &timestamp,
```

Comment:
probably worth it to set StorageResolution to 1 so that we can create high-resolution alarms https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_MetricDatum.html